Write guide on how Data Package compares with CSVW

peterdesmet opened the pull request:
In response to #984, this PR adds a **guide** comparing Data Package with CSV on the Web (CSVW) a set of recommendations published by W3C in 2016 for documenting and accessing CSV files: https://www.w3.org/TR/2016/NOTE-tabular-data-primer-20160225/

It compares the two standards in terms of scope, maintenance, adoption (pending), building on other standards as well as a comparison of all CSVW properties and their equivalent in Data Package.

peterdesmet committed:
Write guide on how Data Package compares with CSVW

peterdesmet commented:
To preview the page, see https://csvw.datapackage-6gp.pages.dev/guides/data-package-csvw/

peterdesmet committed:
Use "compare with" in title

peterdesmet committed:
Format tips as notes

nichtich commented on content/docs/guides/data-package-csvw.md:
```diff
@@ -0,0 +1,191 @@
+---
+title: How does Data Package compare with CSVW
```
title: Comparision with CSVW

nichtich commented on content/docs/guides/data-package-csvw.md:
```diff
@@ -0,0 +1,191 @@
+
+Data Package is a set of specifications to describe datasets, data files and tabular data. It can describe data files that are not tabular and includes extra specifications for those that are (i.e. [Table Dialect](/standard/table-dialect/) and [Table Schema](/standard/table-schema/)).
+
+CSVW is a set of specifications to describe delimited text files (CSV, TSV) only.
```
, in particular the [Model for Tabular Data and Metadata on the Web](https://www.w3.org/TR/tabular-data-model/) and the [Metadata Vocabulary for Tabular Data](https://www.w3.org/TR/tabular-metadata/).

peterdesmet replied:
Thanks, will refer to the versioned URLS + mention it also has two transformation specs (to JSON and to RDF).

nichtich commented on content/docs/guides/data-package-csvw.md:
```diff
@@ -0,0 +1,191 @@
+| [@id](https://www.w3.org/TR/2015/REC-tabular-metadata-20151217/#datatype-id) | Custom property |  |
+| [@type](https://www.w3.org/TR/2015/REC-tabular-metadata-20151217/#datatype-type) | No |  |
+
+### 
```
I think this `###` can be removed.

nichtich commented on content/docs/guides/data-package-csvw.md:
```diff
@@ -0,0 +1,191 @@
+
+Data Package defines its own properties for often-used properties (such as description, contributors, licences, sources), none of which are required. Users can include properties from other standards as [custom properties](/standard/glossary/#custom-properties). It is also possible to add or extend properties by referencing a JSON schema in [@schema](/standard/data-package/#dollar-schema), which  ...
+
+CSVW makes extensive use of other standards (such as [JSON-LD](https://www.w3.org/TR/json-ld/), [XML Schema Datatypes](https://www.w3.org/TR/xmlschema11-2/), Compact URIs) and defines how to use those.
```
In particular, CSVW includes rules how to [connect tabular data with metadata on the Web](https://www.w3.org/TR/tabular-data-model/#locating-metadata) and how to transform data and metadata to RDF.

peterdesmet replied:
I've added this info as a separate section Linking Data with Metadata

nichtich reviewed (changes requested):
Thanks a lot! Some minor additions or suggested changes.

peterdesmet committed:
Update title

Cf @nichtich suggestion

peterdesmet committed:
Reference CSVW specs

peterdesmet committed:
Describe extensibility

peterdesmet committed:
Describe how metadata and data are linked

Cf. @nichtich suggestion

peterdesmet committed:
Fix URLs

peterdesmet committed:
Mention OKFN in maintenance + pull request

Ping @roll @sapetti9

roll committed:
Don't format CSVW guide as it makes tables unreadable

peterdesmet committed:
Describe software and adoption

peterdesmet committed:
Merge branch 'csvw' of https://github.com/frictionlessdata/datapackage into csvw

peterdesmet committed:
First list csvw, then data package

peterdesmet commented:
- I have completed the section on adoption and software: https://csvw.datapackage-6gp.pages.dev/guides/data-package-csvw/#adoption-and-software-support
- In each section, I now mention CSVW first (cf. the tables).
- @nichtich I think I now have addressed your comments

roll committed:
Merge branch 'main' into csvw

peterdesmet committed:
Shorten section title
